Marlin/src/gcode/feature/pause/M125.cpp

```diff
@@ -76,6 +78,8 @@ void GcodeSuite::M125() {
   const float y_pos = parser.linearval('Y')
     #ifdef PAUSE_PARK_Y_POS
       + PAUSE_PARK_Y_POS
+    #else
+      + current_position[Y_AXIS]
     #endif
```

Comment:
If YPOS not defined ,  only x moves
Very usefull when we have a moving bed and our purge bucket /pause area is only on the right , only the carriage need to move